cmd/payload-testing-prow-plugin/server.go

```diff
@@ -199,16 +212,23 @@ func (s *server) handle(l *logrus.Entry, ic github.IssueCommentEvent) string {
 				jobTuples = append(jobTuples, jobTuple)
 			}
 		}
+		specLogger.WithField("duration", time.Since(startResolveTests)).Debug("resolving tests completed")
```

Comment:
Could we log how many jobs did we manage to resolve?

Reply:
Done.